### PR TITLE
APPSRE-8300: Optionally silence successful deploy notices

### PR DESCRIPTION
**Pull request by lisa**
This PR is paired with https://github.com/app-sre/qontract-reconcile/pull/3913

### CODE REVIEW

File: schemas/app-sre/saas-file-2.yml [hunk 1]
```diff
@@ -23,6 +23,11 @@ properties:
   pipelinesProvider:
     "$ref": "/common-1.json#/definitions/crossref"
     "$schemaRef": "/app-sre/pipelines-provider-1.yml"
+  # Setting skipSuccessfulDeployNotifications to true will cause successful
+  # deployments to not have a successful deployment notice sent.
+  skipSuccessfulDeployNotifications:
+    type: boolean
+#    default: false
```
Comment (geoberle) [inline schemas/app-sre/saas-file-2.yml:30]: sadly no default values supported. it is up to the integration code to interpret `null`

**Comment by lisa** [comment]
@geoberle PR is updated with your feedback. Thank you again.

**Review by chassing — APPROVED** [review]

**Comment by OliviaHY** [comment]
LGTM

**Pull request merged**